README.md

```diff
@@ -45,7 +45,7 @@ Since the setup varies a lot for different hardware types, the example configura
 * parallella (Ubuntu)
 * raspberry-pi (Archlinux ARM, Raspbian)
 * raspberry-pi-3 (Archlinux ARM (armv8))
-* raspberry-pi-4 (Archlinux ARM (armv7))
+* raspberry-pi-4 (Archlinux ARM (armv7), Ubuntu 20.04 LTE))
```

Comment:
Shouldn't that be: 
`* raspberry-pi-4 (Archlinux ARM (armv7), Ubuntu 20.04 LTS (armv8))`

Reply:
armv8 and LTS = Long Time Stable... not LTE

Reply:
indeed. I'll fix this...
```suggestion
* raspberry-pi-4 (Archlinux ARM (armv7), Ubuntu 20.04 LTE))
```